lib/Doctrine/Connection.php

```diff
@@ -241,41 +243,31 @@ public function isConnected()
     }
 
     /**
-     * getOptions
-     *
      * Get array of all options
      *
-     * @return void
+     * @return array<string, mixed>
      */
     public function getOptions()
```

Comment:
The is a reason that prevents you to type hint the returns to array?

Reply:
true, I can add it